tools/js-sdk-release-tools/src/common/npmUtils.ts

```diff
@@ -32,10 +39,48 @@ export function getArtifactName(info: NpmPackageInfo) {
     return `${name}-${version}.tgz`;
 }
 
-export async function tryGetNpmView(packageName: string): Promise<{[id: string]: any} | undefined> {
+export async function tryGetNpmView(packageName: string): Promise<{ [id: string]: any } | undefined> {
     try {
         return await fetch.json(`/${packageName}`);
     } catch (err) {
         return undefined;
     }
+}
+
+export async function tryCreateLastStableNpmView(lastStableVersion: string, packageName: string, packageFolderPath: string) {
+    logger.info(`Start to get and clone Api View file from last ${packageName} stable release tag.`);
+    const targentApiViewPath = getApiReviewPath(packageFolderPath).split("sdk");
+    const apiViewPath = path.join("sdk", targentApiViewPath[targentApiViewPath.length - 1]).replace(/\\/g, "/");
+
+    const gitCommand = `git --no-pager show ${packageName}_${lastStableVersion}:${apiViewPath}`;
+
+    try {
+        const lastStableApiView = shell.exec(gitCommand, { silent: true });
+        const lastStableApiViewContext = lastStableApiView.stdout;
+
+        const lastStableApiViewPath = getApiReviewPath(path.join(packageFolderPath, 'changelog-temp', 'package'));
+        fs.writeFileSync(lastStableApiViewPath, lastStableApiViewContext, { encoding: 'utf-8' });
+        logger.info(`Create Api View file from the tag ${packageName}_${lastStableVersion} package successfully`);
+    } catch (error) {
+        logger.error(error);
+        logger.error(`Failed to read Api View file in ${apiViewPath} from the tag ${packageName}_${lastStableVersion}.`);
+    }
+}
+
+export function tryCreateLastChangeLog(packageFolderPath: string, packageName: string, version: string, targetChangelogPath: string) {
```

Comment:
I feel liek you can make one function (with `tryCreateLastStableNpmView`) and reuse it